### ISSUE TITLE
What about not embossed card's?

### ISSUE THREAD

**Issue by dimiroll** [card-io/card.io-dmz#7]
I have tested a lot of card's, but i surprised why card.io doesn't support not embossed (flat) cards?
I would like to do training of CNN myself, but i don't have whole training set. The training one of existed net's with small learning rate is not good idea, i think. 
So, can you  support my begining?

**Comment by dgoldman-ebay** [comment]
@dimiroll thanks for your interest in improving card.io!

> I have tested a lot of card's, but i surprised why card.io doesn't support not embossed (flat) cards?

Due to their many differences (in font-size, typefaces, digit color, lack of shadows, etc.), non-embossed cards require different digit-categorizing models than embossed cards. There is a great deal of variety in non-embossed cards, so supporting them is a lot of work. And non-embossed cards are still rare relative to embossed cards. As a result, so far, it has not been enough of a priority to justify the work that supporting them would require.

> I would like to do training of CNN myself, but i don't have whole training set.

Unfortunately, our training set contains lots of personal credit card images, which we cannot release.

> The training one of existed net's with small learning rate is not good idea, i think. 

We are always open to suggestions. However, we have already experimented with a very wide variety of hyper parameters and model types during training.

**Comment by shaileshnamjoshi** [comment]
Hi David,

Can card io able to scan cards given in the below link.
https://usa.visa.com/dam/VCOM/download/merchants/New_VBM_Acq_Merchant_62714_v5.pdf

**Comment by dgoldman-ebay** [comment]
@shaileshnamjoshi card.io can handle most "traditional" cards (although the presence of the hologram behind some of the numbers, as shown there, might be problematic). However, any non-traditional layouts, such as the "Quick Read Card Design" shown there, will not work with card.io.

**Comment by ericlewis** [comment]
@dgoldman-ebay how would i train with my own images?

**Comment by dgoldman-ebay** [comment]
@ericlewis -- I've responded in your [other issue](https://github.com/card-io/card.io-dmz/issues/26).
